task/03-date-tasks.js

```diff
@@ -94,14 +103,19 @@ function timeSpanToString(startDate, endDate) {
  *    Date.UTC(2016,3,5,21, 0) => Math.PI/2
  */
 function angleBetweenClockHands(date) {
-    throw new Error('Not implemented');
+  let hours =
+    date.getUTCHours() > 12 ? date.getUTCHours() - 12 : date.getUTCHours();
+  let angle = Math.abs(
+    0.5 * (60 * hours + date.getUTCMinutes()) - 6 * date.getUTCMinutes()
+  );
+  let new_angle = angle > 180 ? 360 - angle : angle;
+  return new_angle * (Math.PI / 180);
 }
 
-
 module.exports = {
-    parseDataFromRfc2822: parseDataFromRfc2822,
-    parseDataFromIso8601: parseDataFromIso8601,
-    isLeapYear: isLeapYear,
-    timeSpanToString: timeSpanToString,
-    angleBetweenClockHands: angleBetweenClockHands
+  parseDataFromRfc2822: parseDataFromRfc2822,
+  parseDataFromIso8601: parseDataFromIso8601,
```

Comment:
the PR has a lot of redundant changes in styling, spaces, comments....
Please follow simple rules:
 - do not add any non logic changes to code if review is required
 - if the changes in styling are important - you can do it as separate PR with styling only changes to avoid noise in the main PR

Reply:
Thank you for the review. Can it be problem with prettier plugin in VScode? Because i didn't change anything except deleting throw new Error and writing code.

Reply:
Yes, I guess that autoformat of IDE, but you will be facing with that in future in your projects so that do change that you need understand how to avoid it =)